source/NVDAObjects/IAccessible/MSHTML.py

```diff
@@ -84,6 +85,23 @@ def __getitem__(self,item):
 		self.cache[item]=value
 		return value
 
+	def __contains__(self,item):
+		try:
+			return self.containsCache[item]

```

Comment:
If we do this, the block below that actually runs hasAttribute never gets executed.